messages/src/msg_fields/protocols/present_proof/propose.rs

```diff
@@ -98,27 +99,37 @@ impl Serialize for PresentationPreviewMsgType {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "lowercase")]
+pub enum AttrValue {
+    Plain {
+        #[serde(skip_serializing_if = "Option::is_none")]
+        value: Option<String>,
+    },
+    Encoded {
+        value: String,
+        #[serde(rename = "mime-type")]
+        mime_type: MaybeKnown<MimeType>,
+    },
+}
```

Comment:
This should **not** be duplicated, but rather both attributes should reuse the same `AttrValue` enum. It should be placed somewhere where both protocols have access to. Maybe create a `common` module in `msg_fields` module with an `attr_value` module inside?